ietf/group/tests.py

```diff
@@ -41,6 +41,11 @@ def test_stream_documents(self):
         self.assertEqual(r.status_code, 200)
         self.assertContains(r, draft.name)
 
+        EditorialDraftFactory() # Quick way to ensure RSWG exists.
```

Comment:
Is this needed? Since you have `fetch_redirect_response=False`, I don't think you'll actually hit the view that checks whether the group exists.

Reply:
I was fighting against the internals of `assertRedirects` since I was redirecting to something that redirected. While that might have been better (since the redirect was to the dispatcher for group), I short-circuited to what the current place the dispatcher for group sends rswg.

Reply:
Ok - if you think it was better as it was I don't object to that, just want to be sure that we test that the redirects will eventually land.